### DISCUSSION TITLE
MiniPick and MiniFiles support in sainnhe's themes

### DISCUSSION THREAD

**Discussion by pkazmier** [nvim-mini/mini.nvim#1043] [General]
While my primary themes are all mini.hues, I did spend yesterday (I'm on vacation this week) [tuning](https://github.com/pkazmier/nvim/commit/558c7c518925eb6d514f168d48e1a2a6a9827ec6) all four of sainnhe's colorschemes ([Everforest](https://github.com/sainnhe/everforest), [Gruvbox-Material](https://github.com/sainnhe/gruvbox-material), [Edge](https://github.com/sainnhe/edge), and [Sonokai](https://github.com/sainnhe/sonokai)) to my liking as secondaries themes. In doing so, I noticed a two mini highlight groups that need tweaking in all four themes. Specifically, `MiniPickPrompt` and `MiniFilesFile`. 

I was going to submit PRs to the projects for all four, but I thought it might be a good idea to confirm with you if you had any other tweaks you were planning that these could be queued up with so as to not overwhelm the maintainer. Obviously, I've already fixed these in my configurations, but wanted to make sure other mini users had a good experience out-of-the-box so to speak if they use those themes.

---

`MiniPickPrompt` does not blend with the background of the float:
<img width="1295" alt="image" src="https://github.com/echasnovski/mini.nvim/assets/747855/0104f48e-7d1f-45da-851c-6f03a69b96d7">

`MiniFilesFile` does not look right if the cursor line is enabled:
<img width="1295" alt="image" src="https://github.com/echasnovski/mini.nvim/assets/747855/bc572339-6fab-4e3b-8578-c73049032522">

My fixes for both:
```lua
    set_hl("MiniPickPrompt", palette.blue,   config.float_style == "dim" and palette.bg_dim or palette.bg2)
    set_hl("MiniFilesFile",  palette.fg,     palette.none)
```

**Comment by echasnovski** [comment]
Yep, I agree with both of these changes.

The "MiniPickPrompt" I tried to match the style of "FloatTitle": if in color scheme it has the same background as "NormalFloat", then so should be a "MiniPickPrompt". This is the case at least in 'everforest', so makes sense to adjust "MiniPickPrompt".

The "MiniFilesFile" one I simply missed when playtesting before making PRs. Yes, ideally it should not define background.

(I must have been starting to feel exhaustion at this point when making all those color scheme PRs :sweat_smile:)

**Reply by pkazmier** [reply]
I cannot imagine how tiresome it was updating all of those themes! I spent hours yesterday tuning these four the way I like them. In any case, I've submitted 4 PRs to address the issues I discovered:

- https://github.com/sainnhe/everforest/pull/142
- https://github.com/sainnhe/sonokai/pull/105
- https://github.com/sainnhe/edge/pull/78
- https://github.com/sainnhe/gruvbox-material/pull/210

**Reply by pkazmier** [reply]
These have been merged now.

**Comment by abeldekat** [comment]
`(I must have been starting to feel exhaustion at this point when making all those color scheme PRs 😅)`

That was an epic effort, covering all the themes I use!
